libs/blocks/footer/footer.js

```diff
@@ -150,6 +150,13 @@ export class Footer {
         const ariaExpanded = regionButton.classList.contains('inline-dialog-active');
         regionButton.setAttribute('aria-expanded', ariaExpanded);
       });
+      document.addEventListener('click', (e) => {
+        const ariaExpanded = regionButton.classList.contains('inline-dialog-active');
+        if (ariaExpanded && !e.target.closest('.footer-region')) {
```

Comment:
Nit: This can be shorter:
```suggestion
        if (regionButton.classList.contains('inline-dialog-active') && !e.target.closest('.footer-region')) {
```